packages/eas-cli/src/build/android/configure.ts

```diff
@@ -7,14 +7,15 @@ import { gitAddAsync } from '../../utils/git';
 import { ConfigureContext } from '../context';
 import { isExpoUpdatesInstalled } from '../utils/updates';
 import { configureUpdatesAsync, syncUpdatesConfigurationAsync } from './UpdatesModule';
-import { configureApplicationIdAsync } from './applicationId';
+import { configureApplicationIdAsync, ensureApplicationIdIsValidAsync } from './applicationId';
 
 export async function configureAndroidAsync(ctx: ConfigureContext): Promise<void> {
   if (!ctx.hasAndroidNativeProject) {
     return;
   }
   await AndroidConfig.EasBuild.configureEasBuildAsync(ctx.projectDir);
   await configureApplicationIdAsync(ctx.projectDir, ctx.exp, ctx.allowExperimental);
+  await ensureApplicationIdIsValidAsync(ctx.projectDir);
```

Comment:
Maybe we should also validate the application id in `configureApplicationIdAsync`? 
I mean it's currently possible that the user will be prompted to choose an invalid id.

Reply:
maybe, but it complicates that logic significantly so I decided not to that yet